src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -1,18 +1,25 @@
 package uk.gov.nationalarchives.tdr.api.graphql.fields
 
-import java.util.UUID
+import java.sql.Timestamp
+import java.util.{Date, UUID}
 
 import io.circe.generic.auto._
 import sangria.macros.derive._
 import sangria.marshalling.circe._
-import sangria.schema.{Argument, Field, InputObjectType, IntType, ObjectType, OptionType, StringType, fields}
-import uk.gov.nationalarchives.tdr.api.auth.{ValidateHasExportAccess, ValidateSeries, ValidateUserOwnsConsignment}
+import sangria.schema.{Argument, Field, InputObjectType, IntType, LongType, ObjectType, OptionType, StringType, fields}
+import uk.gov.nationalarchives.tdr.api.auth.{ValidateHasExportAccess, ValidateSeries, ValidateUserHasAccessToConsignment}
 import uk.gov.nationalarchives.tdr.api.graphql._
 import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes._
 
 object ConsignmentFields {
 
-  case class Consignment(consignmentid: UUID, userid: UUID, seriesid: UUID)
+  case class Consignment(consignmentid: UUID,
+                         userid: UUID,
+                         seriesid: UUID,
+                         dateTime: Option[Timestamp],
```

Comment:
This type is `java.sql.Timestamp`. It doesn't seem right to be working with database-specific types at the Sangria level of the code, because this layer shouldn't know anything about the database. I'd have expected this to be a type from `java.time`, and for the Service classes to translate between SQL timestamps and regular date-times.

Reply:
Good point